Sage/carbon
# fix(colors): make used colors in Carbon DLS compliant

### Proposed behaviour
Colors should be compliant with DLS specifications.

### Current behaviour
There is a mismatch between Carbon and DLS colors. Example:

- DLS

![image](https://user-images.githubusercontent.com/47244523/79978307-0020ba00-84a0-11ea-82bb-bea273a1d095.png)

- Carbon:
![image](https://user-images.githubusercontent.com/47244523/79978354-13cc2080-84a0-11ea-893e-52f2a7648ca4.png)

### Checklist


- <del> Storybook added or updated</del>
- <del>Typescript `d.ts` file added or updated</del>

### Additional context
This fixes issue #2889 

### Testing instructions

Confirm that all colors now follow DLS specs. Executing UT's and Cypress e2e tests and check that they pass.
